packages/suite/src/views/suite/error/index.tsx

```diff
@@ -22,7 +22,7 @@ const ErrorPage = () => (
             <Translation id="TR_404_DESCRIPTION" />
         </P>
         <Image image="ERROR_404" />
-        <Link variant="nostyle" target="_self" href="/">
+        <Link variant="nostyle" target="_self" href="/web">
```

Comment:
This is going to work only on production. Please use asset prefix.

I believe something like that is necessary. 
```
const assetPrefix = process.env.ASSET_PREFIX || '';
```

```suggestion
        <Link variant="nostyle" target="_self" href={`/${assetPrefix}`}>
```

It has to work on localhost
http://localhost:8000/
https://suite.trezor.io/web/
https://staging-suite.trezor.io/web
https://suite.corp.sldev.cz/suite-web/develop/web/

error page does not work on sldev now because of wrong configuration (cc @matejkriz) so please test that it works by putting this link to another page just for your testing purposes

Reply:
Thanks for the suggestion. It was a bit of blind fix by me (tested only in product with inspect) because I have somehow broken routing and webusb in web dev mode (only desktop dev mode is fully working for me, was trying to fix it with @matejkriz but so far no luck).

Reply:
Check it again